account_invoice_open_sale/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Open sale order from invoice",
+    "summary": "Show a button on the invoice view to open the related sale order",
+    "version": "12.0.1.0.0",
+    "category": "Sales",
+    "website": "htttps://github.com/oca/sale-worfklow",
```

Comment:
StefanRijnhart 4 hours ago Member

Oops I had this wrong. Should have been sale-financial